docs-chef-io/content/inspec/resources/k8s_pod.md

```diff
@@ -5,59 +5,67 @@ gh_repo = "inspec"
 platform = "k8s"
 
 [menu]
-  [menu.inspec]
-    title = "k8s_pod"
-    identifier = "inspec/resources/k8s/K8s Pod"
-    parent = "inspec/resources/k8s"
+[menu.inspec]
+title = "k8s_pod"
+identifier = "inspec/resources/k8s/K8s Pod"
+parent = "inspec/resources/k8s"
 +++
 
 
-Use the `k8s_pod` Chef InSpec audit resource to test the configuration of...
+Use the `k8s_pod` Chef InSpec audit resource to test the configuration of specific Pods in a namespace
 
 ## Installation
 
 ## Syntax
 
 ```ruby
-describe k8s_pod do
+describe k8s_pods(namespace: "NAMESPACE", name: "NAME") do
   #...
 end
 ```
 
 ## Parameters
 
-`PARAMETER`
-: PARAMETER DESCRIPTION
+`type`
+: type of the K8s resource that is for query
 
-`PARAMETER`
-: PARAMETER DESCRIPTION
+`namespace`
+: namespace of the resource
 
 ## Properties
 
-`PROPERTY`
-: PROPERTY DESCRIPTION
+`uid`
+: UID of the resource
 
-`PROPERTY`
-: PROPERTY DESCRIPTION
+`name`
+: Name of the resource
+
+`namespace`
+: Namespace of the resource
```

Comment:
same goes for namespace, resource_version, kine and metadata

Reply:
ok